tests/image-policy-exists.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Moved this block from tests/test-kustomize-v2.sh as i think it doesn't belong there.